test/integration/scheduler/framework_test.go

```diff
@@ -260,8 +264,8 @@ func TestContextCleanup(t *testing.T) {
 	}
 
 	// Make sure the data inserted in the plugin context is removed.
```

Comment:
Comments need to be updated synchronously.

Reply:
Sorry, I don't understand what is needed here. Can you elaborate please?